Modules/Quotas/test/Quotas.Application.Tests/Tests/Tiers/GetTierById/HandlerTests.cs

```diff
@@ -47,15 +48,13 @@ public async void Gets_tier_by_id_multiple_quotas()
         var tierName = "some-tier-name";
         var tier = new Tier(tierId, tierName);
 
-        var metric1 = new Metric(MetricKey.NumberOfSentMessages, "Number Of Sent Messages");
-        var metric2 = new Metric(MetricKey.FileStorageCapacity, "File Storage Capacity");
-        var metric3 = new Metric(MetricKey.NumberOfRelationships, "Number Of Relationships");
-        var metrics = new List<Metric> { metric1, metric2, metric3 };
+        var metricWithTwoQuotas = new Metric(MetricKey.NumberOfSentMessages, "Number Of Sent Messages");
+        var metricWithOneQuota = new Metric(MetricKey.FileStorageCapacity, "File Storage Capacity");
+        var metrics = new List<Metric> { metricWithTwoQuotas, metricWithOneQuota };
 
-        tier.CreateQuota(metric1.Key, 5, QuotaPeriod.Month);
-        tier.CreateQuota(metric2.Key, 10, QuotaPeriod.Day);
-        tier.CreateQuota(metric3.Key, 12, QuotaPeriod.Week);
-        tier.CreateQuota(metric1.Key, 3, QuotaPeriod.Year);
+        tier.CreateQuota(metricWithTwoQuotas.Key, 5, QuotaPeriod.Month);
+        tier.CreateQuota(metricWithOneQuota.Key, 10, QuotaPeriod.Day);
+        tier.CreateQuota(metricWithTwoQuotas.Key, 3, QuotaPeriod.Year);
```

Comment:
You should use simpler values:

```suggestion
        tier.CreateQuota(metricWithTwoQuotas.Key, 1, QuotaPeriod.Day);
        tier.CreateQuota(metricWithOneQuota.Key, 1, QuotaPeriod.Day);
        tier.CreateQuota(metricWithTwoQuotas.Key, 1, QuotaPeriod.Day);
```

This is a general principle. Only vary test data that actually matters for the outcome of the test. Otherwise the reader wonders whether the variation influences the outcome of the test.